dept44-starter-test/src/main/java/se/sundsvall/dept44/test/AbstractAppTest.java

```diff
@@ -236,6 +247,14 @@ public AbstractAppTest sendRequestAndVerifyResponse(final MediaType mediaType) {
 		if (this.expectedResponseBodyIsNull) {
 			assertThat(this.responseBody).isNull();
 		}
+
+		await()
```

Comment:
Default wait should be 0. If not explicitly set no wait will occure.

Reply:
Added 0 as initial delay. Wich will cause the evaluation to be executed immediately.
Also changed the exception-handling in verify-stubs. (Necessary for async-verifications)